app-backend/common/src/main/resources/reference.conf

```diff
@@ -30,6 +30,7 @@ awsbatch {
   ingestJobQueue = ${?BATCH_JOB_QUEUE_INGEST}
 
   importJobName = ${?BATCH_JOB_NAME_PROCESS_UPLOAD}
+  geojsonImportJobName = ${?BATCH_JOB_NAME_PROCESS_GEOJSON_UPLOAD}
```

Comment:
I needed to add a value for this to the environment to get the api server to start -- probably not worth addressing here, but maybe worth a note/update to .env/notice to the channel in the "add a batch job" issue

Reply:
I did update the `.env.template` file.
I guess I need to update things in the deployment repo / tfvars as well?

Reply:
Yeah, if you don't provide a default then services will fail to start until the deployment repo is updated to include this new environment variable.

Reply:
added a default "placeholder" value so things start. the endpoint will fail to actually initialize the batch job, but that's fine since it doesn't exist.

Reply:
Ah I must have forgotten about it when moving from file review -> testing instructions. I just looked at it and remember seeing it before but didn't make the connection that I'd need to do something with it.